pkg/mocai/helper/helper.go

```
@@ -0,0 +1,41 @@
+
+	for i := 0; i < 12; i++ {
+		aadhaar += strconv.Itoa(rand.Intn(10))
+	}
+	return aadhaar
+}
+
+// Function to generate a random PAN number
+func GenerateRandomPAN() string {
+	var letters = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	var digits = "0123456789"
+	rand.Seed(time.Now().UnixNano())
```

golangci-lint (1.62.2)

SA1019: rand.Seed has been deprecated since Go 1.20 and an alternative has been available since Go 1.0: As of Go 1.20 there is no reason to call Seed with a random value. Programs that call Seed with a known value to get a specific sequence of results should use New(NewSource(seed)) to obtain a local random generator. 

(staticcheck)